state-sync/diem-data-client/src/diemnet/state.rs

```diff
@@ -118,14 +122,34 @@ impl PeerStates {
     }
 
     pub fn update_score_success(&mut self, peer: PeerNetworkId) {
-        self.inner.entry(peer).or_default().update_score_success()
+        let old_score = self.inner.entry(peer).or_default().score;
+        self.inner.entry(peer).or_default().update_score_success();
+        let new_score = self.inner.entry(peer).or_default().score;
+        if old_score <= IGNORE_PEER_THRESHOLD && new_score > IGNORE_PEER_THRESHOLD {
+            debug!(
+                (LogSchema::new(LogEntry::PeerStates)
+                    .event(LogEvent::PeerNoLongerIgnored)
+                    .message(&format!("Peer {:?} will no longer be ignored", peer))
```

Comment:
Likewise, can we not use the `.peer(...)` entry to log the peer.

Reply:
Used .peer field.